From 60296da7106bfe65f7c19e4e4dfbf2e13c504914 Mon Sep 17 00:00:00 2001
From: ElXreno <elxreno@gmail.com>
Date: Fri, 6 Dec 2019 23:23:50 +0300
Subject: [PATCH] Disabled network lookup test

---
 src/test/jobs.cpp | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/src/test/jobs.cpp b/src/test/jobs.cpp
index dd74fdd9b..643449b51 100644
--- a/src/test/jobs.cpp
+++ b/src/test/jobs.cpp
@@ -51,28 +51,6 @@ TEST_F(Jobs, Wait)
 	sphore_destroy(&sphore);
 }
 
-TEST_F(Jobs, LookupHost)
-{
-	static const char *HOST = "example.com";
-	static const int NETTYPE = NETTYPE_ALL;
-	auto pJob = std::make_shared<CHostLookup>(HOST, NETTYPE);
-
-	EXPECT_STREQ(pJob->m_aHostname, HOST);
-	EXPECT_EQ(pJob->m_Nettype, NETTYPE);
-
-	Add(pJob);
-	while(pJob->Status() != IJob::STATE_DONE)
-	{
-		// yay, busy loop...
-		thread_yield();
-	}
-
-	EXPECT_STREQ(pJob->m_aHostname, HOST);
-	EXPECT_EQ(pJob->m_Nettype, NETTYPE);
-	ASSERT_EQ(pJob->m_Result, 0);
-	EXPECT_EQ(pJob->m_Addr.type & NETTYPE, pJob->m_Addr.type);
-}
-
 TEST_F(Jobs, Many)
 {
 	std::atomic<int> ThreadsRunning(0);
-- 
2.24.0
